modules/gtk/call_window.c

```diff
@@ -396,6 +395,9 @@ struct call_window *call_window_new(struct call *call, struct gtk_mod *mod,
 	GtkWidget *button_box, *vbox, *hbox;
 	GtkWidget *duration;
 	int err = 0;
+	err = mtx_init(&last_data_mut, mtx_plain) != thrd_success;
+	if (err)
```

Comment:
please set `err` to `ENOMEM` here

Reply:
sorry, I should have been more clear..

curly brackets are missing. Example code here:

https://github.com/baresip/baresip/blob/main/modules/coreaudio/player.c#L106

Reply:
Saw the example code when I looked for other occurrences of ENOMEN.

Thanks for finding the missing curly brackets. I am a little embarrassed, that this really just happened.